### ISSUE TITLE
Can't login using latest docker release LABEL version="2.1.9"

### ISSUE THREAD

**Issue by stephbaxter** [apioo/fusio#444]
I have pulled the latest docker config (LABEL version="2.1.9"). When trying to access the /apps/fusio/#!/login page (says 1.0.1 at the top ) I get the response

{
    "error": "invalid_client",
    "error_description": "Unknown credentials"
}

Is this related to an earlier issue from Feb which talks about v2.x being incompatible with backed v 1.0.1

Is there a fix for this other than hand installing it?

Thx

**Comment by chriskapp** [comment]
Hi, so currently the "latest" tag contains the 3.0 release candidate version, I would recommend to explicit use the "2.1.9" or "2.1" version to use the latest stable version. But in general the latest version should also work, do you have created a complete fresh installation or is this a migration from another version? Also the credentials which you have entered are correct?

**Comment by stephbaxter** [comment]
Hi Chris

Thanks for the quick reply. I solved my problem. I had created the container with the default password then edited the docker-compose.yml file, deleted the fusiu container (and mysql container) and recreated them BUT did not delete the ./db/ directory so I guess the default backend password was written to the mysql files and not overwritten on recreation of the fusio container so when I tried to login with the new password it was looking for the default one!

It looks like a great systen btw. Thx
